parser/src/main/java/com/plhal/ares/parser/ParserRepositoryImpl.java

```diff
@@ -67,19 +66,25 @@ public Firma najdiFirmu(String ico) {
             return null;
         }
 
-        return Firma.builder()
-                .clenoveStatutarnihoOrganu(pridejClenyStatutarnihoOrganu(doc))
+        Firma firma = Firma.builder()
+                .ico(ico)
+                .clenoveStatutarnihoOrganu(pridejClenyStatutarnihoOrganu(doc, ico))
                 .zakladniKapital(zjistiZakladniKapital(doc))
                 .nazevFirmy(zjistiNazev(doc))
                 .sidlo(zjistiSidlo(doc))
                 .predmetPodnikani(pridejPredmetyPodnikani(doc))
                 .pravniForma(zjistiPravniFormu(doc))
                 .build();
+        for (StatutarniOrgan statOrgan : firma.getClenoveStatutarnihoOrganu()) {
+            statOrgan.setFirma(firma);
+        }
+
+        return firma;
 
     }
 
     // Získávám údaje o členech SU
-    private @NonNull List<StatutarniOrgan> pridejClenyStatutarnihoOrganu(@NonNull Document doc) {
+    private @NonNull List<StatutarniOrgan> pridejClenyStatutarnihoOrganu(@NonNull Document doc, String ico) {
```

Comment:
To `ico` je tu nejaky pozustatek, ze? V metode se nikde nepouziva.

Reply:
Ano to tam zustalo z minula, dal jsem pryc.